static/src/javascripts/projects/common/modules/commercial/dfp/apply-creative-template.js

```diff
@@ -75,10 +75,8 @@ define([
         var creativeConfig = fetchCreativeConfig();
 
         if (creativeConfig) {
-            return Promise.all([
```

Comment:
If the iframe doesn't need to be hidden, this operation is just wasting a frame, really. So we can happily leave it be.